icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -50,17 +55,22 @@
 #
 # b) CLDR-related variables
 #
-# CLDR_DIR:      Path to root of CLDR sources, below which are the common and
-#                tools directories.
+# CLDR_DIR:      For most of the process, this is the path to the to root of
+#                standard CLDR sources, below which are the common and
+#                tools directories. For running LdmlConverter, this needs to be
+#                temporarily reset to the parallel root for the production data,
+#                corresponding to $CLDR_TMP_DIR/production (see description of
+#                CLDR_TMP_DIR below).
 # CLDR_CLASSES:  Path to the CLDR Tools classes directory. If not set, defaults
 #                to $CLDR_DIR/tools/java/classes
 #
 # CLDR_TMP_DIR:  Parent of temporary CLDR production data.
```

Comment:
Seems like we want a bug raised to get rid of this.

Reply:
Yes, filed https://unicode-org.atlassian.net/browse/ICU-20998 for this